exporter/lokiexporter/config_test.go

```diff
@@ -15,515 +15,84 @@
 package lokiexporter
 
 import (
-	"path/filepath"
 	"testing"
-	"time"
 
-	"github.com/prometheus/common/model"
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-	"go.opentelemetry.io/collector/component/componenttest"
-	"go.opentelemetry.io/collector/config"
 	"go.opentelemetry.io/collector/config/confighttp"
-	"go.opentelemetry.io/collector/config/configtls"
-	"go.opentelemetry.io/collector/exporter/exporterhelper"
-	conventions "go.opentelemetry.io/collector/semconv/v1.6.1"
-	"go.opentelemetry.io/collector/service/servicetest"
 )
 
-func TestLoadConfig(t *testing.T) {
```

Comment:
Let's maybe keep it the `TestLoadConfig` for the new exporter. We can also use `confmaptest` 
 https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/13224 to read the config file.